packages/logging/src/__tests__/environment.test.ts

```
@@ -0,0 +1,186 @@
+      process.env.NODE_ENV = 'development'
+      expect(detectEnvironment()).toBe('development')
+
+      process.env.NODE_ENV = 'dev'
+      expect(detectEnvironment()).toBe('development')
+    })
+
+    it('should default to development for unknown values', () => {
+      process.env.NODE_ENV = 'staging'
+      expect(detectEnvironment()).toBe('development')
+
+      delete process.env.NODE_ENV
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)